x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.ts

```diff
@@ -278,7 +278,7 @@ export const PARAMETERS_DEFINITION = {
             i18n.translate(
               'xpack.idxMgmt.mappingsEditor.parameters.validations.copyToIsRequiredErrorMessage',
               {
-                defaultMessage: 'Specify a copy value.',
+                defaultMessage: 'Copy to is required.',
```

Comment:
This should be "Destination field name is required" or "Group field name is required" if we adopt my suggestion above.

Reply:
Good catch!